spec/ssh_scan/banner_spec.rb

```diff
@@ -84,6 +93,9 @@
       "SSH-2.0-ros" => {
         :os_class => SSHScan::OS::ROS,
       },
+      "SSH-1.99-DOPRA" => {
```

Comment:
Ah, you are right, since this is merged, can you file an issue for it so we don't lose it?